app.js

```diff
@@ -10,3 +10,151 @@ document.querySelectorAll('.menu-item').forEach((n) => n.addEventListener('click
+    featured_image: 'image/styling.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/rmjspecial',
+    live_demo: 'https://github.com/rmjspecial/personal-portfolio',
+  },
+  Fitness: {
+    title: 'Fitness',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'image/fitness.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/rmjspecial',
+    live_demo: 'https://github.com/rmjspecial/personal-portfolio',
+  },
+};
+
+// reverse the order of work card
+function workCardShouldReverse(index) {
+  if (index % 2 === 0) {
+    return 'row-reverse';
+  }
+  return '';
+}
+
+// create the breadcrumbs
+function breadcrumbsHtml(workObj, key, bread = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const breadcrumb in workObj[key].breadcrumbs) {
+    if (breadcrumb) {
+      bread += `<li>${workObj[key].breadcrumbs[breadcrumb]}</li>`;
+    }
+  }
+  /* eslint-enable no-restricted-syntax */
+
+  return bread;
+}
+
+// create the language list
+function languageHtml(workObj, key, langList = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const lang in workObj[key].lang_list) {
+    if (lang) {
+      langList += `<li><span>${workObj[key].lang_list[lang]}</span></li>`;
+    }
+  }
+
+  return langList;
+}
+
+//  select the works container
+const workCardContainer = document.querySelector('.works-grid-container');
+let workCard = '';
+let index = 0;
+
+// create the work cards
+for (const key in works) {
+  if (key) {
+    index += 1;
+    workCard
+      += `
+        <div class="flex works-grid-item ${workCardShouldReverse(index)}">
```

Comment:
- [ ] Please make sure the project technologies and buttons are aligned on the right side of the card along with the project description and it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174289259-4ef3aeba-feee-4e9c-a663-3233837ffc2d.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/174289301-fd3eba7c-0824-4eb6-9a10-16f678fb2233.png)